runloop.c

```diff
@@ -3615,9 +3727,19 @@ bool runloop_init_libretro_symbols(
 
                if (string_is_empty(path))
                {
-                  RARCH_ERR("[Core]: Frontend is built for dynamic libretro cores, but "
+                  const char* content_path = path_get(RARCH_PATH_CONTENT);
+                  if (!string_is_empty(content_path))
+                  {
```

Comment:
Can you get rid of the  brackets here when it's just a single line if conditional?

Reply:
Heartbleed </3 I sure can, though - is there a style guide/clangformat/etc I can follow?